app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<header>Sign ups</header>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <%= f.text_field :Name, autofocus: true, autocomplete: "Name", placeholder: "Name" %>
+  <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+  <%= f.password_field :password, autocomplete: "new-password", placeholder: "Password (min 6 characters)"  %>
+  <%= f.password_field :password_confirmation, autocomplete: "new-password", placeholder: "Confirm password" %>
+
+
+  <div class="greenButton">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<div class="headerBack">
+  <%= link_to "<", users_path	 %>
+</div>
+
+<%= render "users/shared/links" %>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219305949-ec3e4328-8539-4834-b9c2-707c1248b1f6.png)

**[Optional]**

 You have a horizontal scroll on your signup page which is not professional and following the design at all :thinking: Kindly fix this issue and follow the design guidelines to make your project look professional :+1: